src/main/java/by/onliner/cucumber/steps/ShoppingCartSteps.java

```diff
@@ -49,4 +51,52 @@ public void checkedEmptyCartMessage(String expectedMessage) {
                 String.format("The message [%s] wasn't contains at expected empty cart message [%s]",
                         emptyCartMassage, expectedMessage));
     }
+
+    @When("Get item name")
+    public void getItemName() {
+        ScenarioContext.getScenarioContext().setContext(Context.NAME, catalogPageItem.getItemName());
```

Comment:
I would recommend to specify name of what you are setting there as in the future you are going to have many different names

Reply:
Renamed